sphinx_gallery/binder.py

```diff
@@ -64,8 +64,8 @@ def gen_binder_url(fpath, binder_conf, gallery_conf):
     path_link = path_link.replace(os.path.sep, '/')
 
     # Create the URL
-    binder_url = binder_conf['url']
-    binder_url = '/'.join([binder_conf['url'],
+    binder_url = binder_conf['binderhub_url']
```

Comment:
For backward compat perhaps you should detect the presence of the `url` key, emit a warning, and put it in `binderhub_url`?